src/components/implementation/no_interface/llbooter/boot_deps.h

```diff
@@ -9,10 +9,20 @@
 #include <res_spec.h>
 #include <ps.h>
 #include <bitmap.h>
+#include <fault_handler.h>
 
 /* Assembly function for sinv from new component */
 extern word_t hypercall_entry_rets_inv(spdid_t cur, int op, word_t arg1, word_t arg2, word_t arg3, word_t *ret2, word_t *ret3);
 
+/* Assembly function for sinv for the fault handlers */
```

Comment:
consider defining unsigned longs as something else in the future, perhaps ptrint_t. This is for the long term.

Reply:
Why aren't they just `word_t`. That is pretty architecture specific, isn't? You're also using that for registers among others so!